facia/test/FaciaControllerTest.scala

```diff
@@ -101,4 +102,26 @@ import services.ConfigAgent
     val usResult = FaciaController.renderFront("technology")(usRequest)
     header("Location", usResult).head should endWith ("/us/technology")
   }
+
+  it should "understand the international edition" in {
+
+    Switches.InternationalEditionSwitch.switchOn()
+
+    val control = FakeRequest("GET", "/").withHeaders("X-GU-Edition" -> "INTL", "X-GU-International" -> "control")
+    val redirectToUk = FaciaController.renderFront("")(control)
+    header("Location", redirectToUk).head should endWith ("/uk")
+
+    val international = FakeRequest("GET", "/").withHeaders("X-GU-Edition" -> "INTL", "X-GU-International" -> "international")
+    val redirectToInternational = FaciaController.renderFront("")(international)
+    header("Location", redirectToInternational).head should endWith ("/international")
+
+    val ukOptInRequest = FakeRequest("GET", "/").withHeaders("X-GU-Edition" -> "UK", "X-GU-International" -> "international")
+    val redirect = FaciaController.renderFront("")(international)
+    header("Location", redirect).head should endWith ("/uk")
```

Comment:
I added this breaking test which I think proves there is something not quite right with the `isInternational` logic.

Anyone who chooses a different edition should get that edition in preference to the international edition.